package.json

```diff
@@ -41,7 +41,11 @@
     "test:integration": "playwright test --config ./integration/playwright.config.ts",
     "posttest:integration": "rimraf ./.tmp/integration",
     "test:primary": "jest",
+    "changeset": "changeset",
```

Comment:
Keeping the original scripts around as a fallback for now. I want to use the Changesets GitHub action to handle next week's release, but if anything goes awry we can still easily release manually.